azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/configuration/LocalizationConfiguration.java

```diff
@@ -28,53 +28,55 @@
  * @see CallComposite
  */
 public class LocalizationConfiguration {
-    private final String languageCode;
+    private final LanguageCode languageCode;
     private int layoutDirection;
 
     /**
      * Create Localization configuration.
      *
-     * @param languageCode    string eg,. "en"
+     * Set {@link LanguageCode};.
+     *
+     * @param languageCode    The {@link LanguageCode}; eg,. LanguageCode.fromString("en")
      * @param layoutDirection layout direction eg,. LayoutDirection.RTL
      */
-    public LocalizationConfiguration(final String languageCode, final int layoutDirection) {
+    public LocalizationConfiguration(final LanguageCode languageCode, final int layoutDirection) {
         this.languageCode = languageCode;
         this.layoutDirection = layoutDirection;
     }
 
     /**
      * Create Localization configuration.
      *
-     * @param languageCode string eg,. "en"
+     * @param languageCode The {@link LanguageCode}; eg,. LanguageCode.fromString("en")
      */
-    public LocalizationConfiguration(final String languageCode) {
+    public LocalizationConfiguration(final LanguageCode languageCode) {
         this.languageCode = languageCode;
     }
 
     /**
-     * Get current language String.
+     * Get current LanguageCode enum
      *
-     * @return language string
+     * @return The {@link LanguageCode};
      */
-    public String getLanguageCode() {
+    public LanguageCode getLanguageCode() {
         return languageCode;
     }
 
     /**
-     * Get isRightToLeft boolean.
+     * Get layoutDirection int.
      *
-     * @return isRightToLeft boolean
+     * @return layoutDirection int
      */
     public int layoutDirection() {
         return layoutDirection;
     }
 
     /**
-     * Get supported Locale string
+     * Get supported Locale list
      *
-     * @return List of supported Locale as String List
+     * @return List of supported languageCode as {@link LanguageCode}; List
      */
-    public static List<String> getSupportedLanguages() {
+    public static List<LanguageCode> getSupportedLanguages() {
```

Comment:
This method should return a read only collection

Reply:
Changed accordingly. Thanks :)